src/main/java/frc/robot/Robot.java

```diff
@@ -15,6 +17,7 @@ public class Robot extends TimedRobot {
 
     private ClawGripperSystem clawGripperSystem;
     private ClawJointSystem clawJointSystem;
+    private ArmJointSystem armJointSystem;
```

Comment:
initialize this system and create the command and set it as default command.

Make sure to keep the command instance in a variable so it's methods could be accessed.